### PR TITLE
Clean-up Profile Components

### PR DESCRIPTION
**Pull request by leekahung**
## Clean-up Profile Components

The overall aim of this PR is simply to go through the existing codebase and cleaning up the existing component files. This one specifically will target the components in `src/components/Profile`. I aim to subsequently update the other component files in their own PRs.

---
## This PR:
**1.**    Cleans up codebase in `src/components/Profile`
**2.**   Cleans up typedef in `src/typedefs.js`

---
### Components
- `src/components/Profile/ProfileComponent.jsx`
- `src/components/Profile/ProfileEditButtonGroup.jsx`
- `src/components/Profile/ProfileImageField.jsx`
- `src/components/Profile/ProfileImageField.jsx`

### Other Files
- `src/typedefs.js`

---
## Future Steps/PRs Needed to Finish This Work (optional):
- Additional steps include cleaning up other files within `src/components/`

### CODE REVIEW

File: src/components/Profile/ProfileComponent.jsx [hunk 1]
```diff
@@ -129,26 +129,22 @@ const ProfileComponent = ({ contactProfile }) => {
...
             edit={edit}
-            disabled={contactProfile}
           />
           <Box sx={{ display: 'flex', gap: '10px' }}>
             <Typography>Date of Birth: </Typography>
             {edit ? (
               <FormControl>
                 <LocalizationProvider dateAdapter={AdapterDayjs}>
                   <DatePicker
-                    name="date"
```
Comment (xscottxbrownx) [inline src/components/Profile/ProfileComponent.jsx:147]: replace one of these deleted attributes with `disableFuture` : https://mui.com/x/api/date-pickers/date-picker/

File: src/components/Profile/ProfileComponent.jsx [hunk 2]
```diff
@@ -129,26 +129,22 @@ const ProfileComponent = ({ contactProfile }) => {
             }
             setInputValue={setProfileName}
             edit={edit}
-            disabled={contactProfile}
```
Comment (xscottxbrownx) [inline src/components/Profile/ProfileComponent.jsx:132]: do we want to be able to edit a `contact`'s profile info?  
Or do you feel these are unnecessary due to the `EDIT` button being conditionally rendered?

The `EDIT` button still rendered when I clicked on a contact's profile link from the contact table.
It shows their webID at top, but all my info below (don't remember if we already discussed this or had issue for it.)

File: src/components/Profile/ProfileComponent.jsx (hunk 1 above)
Comment (xscottxbrownx) [reply src/components/Profile/ProfileComponent.jsx:147]: Right now you can choose future dates in DOB field.

**Review by xscottxbrownx — CHANGES_REQUESTED** [review]

File: src/components/Profile/ProfileComponent.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Profile/ProfileComponent.jsx:147]: Got `disableFuture` included.

File: src/components/Profile/ProfileComponent.jsx (hunk 2 above)
Comment (leekahung) [reply src/components/Profile/ProfileComponent.jsx:132]: Yeah, I think we've discussed this before. It's related to the bug mentioned in Issue #386 that still needs to be patched up. I believe the video example shown there is something that you've encountered here.

Comment (leekahung) [reply src/components/Profile/ProfileComponent.jsx:132]: Created a new PR to patch up this issue for good, see PR #402.

**Review by xscottxbrownx — APPROVED** [review]

**Pull request merged**